modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -843,11 +813,17 @@ There are no valid index configuration values for the constraint-backing range i
 === Handling existing constraints when creating a constraint
 
 Creating an already existing constraint will fail.
+This includes the following scenarios:
+
+* Creating a constraint identical to an already existing constraint.
+* Creating a constraint with a different name but on the same schema as an already existing constraint.
```

Comment:
```suggestion
* Creating a constraint with a different name but on the same constraint type and schema as an already existing constraint. For property type constraints the property type also needs to be the same.
```
if we want to cover the cases where it's considered 'the same' constraint but just with different names...